scripts/lua/cors.lua

```diff
@@ -25,25 +25,28 @@ local _M = {}
 local request = require "lib/request"
 
 function _M.processCall(resourceConfig)
-  if resourceConfig.cors ~= nil then
-    _M.setCorsHeaders(resourceConfig.cors.origin, resourceConfig.cors.methods)
-    if ngx.req.get_method() == "OPTIONS" then
-      request.success(200)
-    end
+  ngx.var.cors_origins = resourceConfig.cors.origin
+  ngx.var.cors_methods = resourceConfig.cors.methods
```

Comment:
I think these two should go inside the if statement below because it will fail if `resourceConfig.cors` is not defined